src/scss/layout/_cta.scss

```diff
@@ -18,7 +18,7 @@
   --stacked-margin-block: var(--shim);
   --title-action-margin-block-end: var(--quarter-shim);
 
-  background-image: url('/assets/media/callout-bg-1.webp');
```

Comment:
are we using this background image for anything still? If not, we can remove it.